frontend/javascripts/oxalis/api/api_latest.ts

```diff
@@ -2257,8 +2258,12 @@ class DataApi {
       layerName,
     ).name;
 
-    if (Store.getState().localSegmentationData[effectiveLayerName].meshes[segmentId] != null) {
+    if (Store.getState().localSegmentationData[effectiveLayerName].meshes?.[segmentId] != null) {
       Store.dispatch(updateMeshVisibilityAction(effectiveLayerName, segmentId, isVisible));
+    } else {
+      throw new Error(
+        `Mesh for segment ${segmentId} was not found in OxalisState.localSegmentationData.`,
```

Comment:
The error was a little cryptic before introducing the check. It failed in the annotation reducer, where the update method threw an error about `object` being undefined. So I thought it would be clearer this way.